### PR TITLE
Build middle section

### PR DESCRIPTION
**Pull request by sevinchek**
In this branch I:
- Added the Works section
- Added the About section
- Update the README file
- Added Screenshots from each section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -62,6 +61,122 @@ <h3>I'am a Software Developer</h3>
                 </a>
             </li>
         </ul>
-    </section>
+      </main>
+      <section>
```
Comment (mricanho) [inline index.html:65]: - [ ] This is the main container, so you should "display: grid" here.

File: index.html [hunk 2]
```diff
@@ -62,6 +61,122 @@ <h3>I'am a Software Developer</h3>
...
+            <h4>Profesional Art<br>Printing Data</h4>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="build-with">
+              <span class="html">html</span>
+              <span class="bootstrap">bootstrap</span>
+              <span class="ruby">Ruby</span>
+            </div>
+            <button>See Project</button>
+          </article>
+        </div>
```
Comment (mricanho) [inline index.html:141]: - [ ] The list of projects should use a `ul` tag to group them, and `li` per project

File: index.html [hunk 3]
```diff
@@ -62,6 +61,122 @@ <h3>I'am a Software Developer</h3>
...
+            <h4>Profesional Art<br>Printing Data</h4>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="build-with">
+              <span class="html">html</span>
+              <span class="bootstrap">bootstrap</span>
+              <span class="ruby">Ruby</span>
+            </div>
+            <button>See Project</button>
+          </article>
+          <article class="prof-art">
```
Comment (mricanho) [inline index.html:111]: - [ ] [OPTIONAL] Article it's not a good tag for this container, you can use just div here and in the other projects

File: index.html [hunk 4]
```diff
@@ -62,6 +61,122 @@ <h3>I'am a Software Developer</h3>
...
+            <h4>Profesional Art<br>Printing Data</h4>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="build-with">
+              <span class="html">html</span>
+              <span class="bootstrap">bootstrap</span>
+              <span class="ruby">Ruby</span>
+            </div>
```
Comment (mricanho) [inline index.html:118]: - [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`

File: index.html [hunk 5]
```diff
@@ -62,6 +61,122 @@ <h3>I'am a Software Developer</h3>
...
+            <button>See Project</button>
```
Comment (mricanho) [inline index.html:139]: - [ ] The button (e.g., "See project") is a link `a` or `button` tag, and follow the button styling: Active, Hover, and Pressed (see design)

File: index.html [hunk 6]
```diff
@@ -62,6 +61,122 @@ <h3>I'am a Software Developer</h3>
...
+          </article>
+        </div>
+      </section>
+      <div class="about">
```
Comment (mricanho) [inline index.html:143]: - [ ] This tag should be another "section" not a div

File: index.html [hunk 7]
```diff
@@ -62,6 +61,122 @@ <h3>I'am a Software Developer</h3>
...
+        <h4>About me</h4>
+        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <button>Get my Resume</button>
```
Comment (mricanho) [inline index.html:146]: - [ ] The button (e.g., "Get my resume") is a link `a` or `button` tag, and follow the button styling: Active, Hover and Pressed (see design)

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @sevinchek, 

Good job so far! 🔥🔥 Actually your portfolio it's very good, almost identical to the original design, excellent work. Your README file, gitflow and commit messages are super professionals.

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


that's how you do it

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jubaan — APPROVED** [review]
Hi @sevinchek , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**